RELEASE_NOTES.md

```diff
@@ -0,0 +1,3 @@
+#### New in 1.1.0 (Released 2018/03/15)
+* enhance build process with fake+paket
```

Comment:
can it mention the "Add dependency management support from @mchaloupka and @michalkovy" please?

Reply:
done + mine